pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/fwd/MultiValueVarByteRawIndexCreator.java

```diff
@@ -119,4 +117,55 @@ public void close()
       throws IOException {
     _indexWriter.close();
   }
+
+  /**
+   * The actual content in an MV array is prepended with 2 prefixes:
+   * 1. elementLengthStoragePrefix - bytes required to store the length of each array element
+   * 2. numberOfElementsStoragePrefix - Number of elements in the array
+   *
+   * This function returns the total bytes needed to store (1) elementLengthStoragePrefix
+   */
+  public static int getElementLengthStoragePrefix(int maxNumberOfElements) {
+    return Integer.BYTES * maxNumberOfElements;
+  }
+
+  /**
+   * The actual content in an MV array is prepended with 2 prefixes:
+   * 1. elementLengthStoragePrefix - bytes required to store the length of each array element
+   * 2. numberOfElementsStoragePrefix - Number of elements in the array
+   *
+   * This function returns the bytes needed to store (2) numberOfElementsStoragePrefix
+   */
+  public static int getNumberOfElementsStoragePrefix() {
```

Comment:
nit: Same as `getElementLengthStoragePrefix`, perhaps modify the function name to indicate it returns a size in bytes.
Just calling it "StoragePrefix" makes me think that this is going to return the prefix itself.